src/library/CMakeLists.txt

```diff
@@ -2,10 +2,7 @@
 vtk_encode_string(
   INPUT "${CMAKE_SOURCE_DIR}/resources/logo32.png"
   NAME F3DIcon
-  HEADER_OUTPUT F3D_ICON_HEADER
-  SOURCE_OUTPUT F3D_ICON_SOURCE
-  BINARY
-  NUL_TERMINATE)
+  BINARY)
```

Comment:
not needed anymore ?

Reply:
`NUL_TERMINATE` was useless, and other variable definition can be deduced.